pkg/querier/queryrange/codec_test.go

```diff
@@ -1010,7 +1012,9 @@ var (
 			"execTime": 22,
 			"linesProcessedPerSecond": 23,
 			"queueTime": 21,
-			"subqueries": 1,
+			"shards": 0,
+            "splits": 0,
+			"subqueries": 0,
```

Comment:
1. `splits` is not formatted correctly looks like.
2. I would assume there shouldn't be `subqueries` keys here.

Reply:
subqueries still exists because it is deprecated but not removed.